mne/io/nirx/tests/test_nirx.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+# Authors: Robert Luke  <mail@robertluke.net>
+#          simplified BSD-3 license
+
+import os.path as op
+
+import numpy as np
+
+from mne.io import read_raw_nirx
+from mne.utils import run_tests_if_main
+from mne.datasets.testing import data_path, requires_testing_data
+from mne.utils import logger, requires_pandas
+
+
+@requires_pandas
+@requires_testing_data
+def test_nirx():
```

Comment:
I would create a dummy `io.RawArray` and compare against it.

Reply:
You would do it that way? Or I must do it that way? I took a look at exima and curry readers and followed their approach.

* Why would your approach of doing it via dummy be better?
* Can you provide an example test for me to work from?

Reply:
I think it's okay as is. It's explicit, which is nice.